assets/js/modules/analytics/components/dashboard/DashboardAllTrafficWidget/UserCountGraph.js

```diff
@@ -111,10 +112,12 @@ export default function UserCountGraph( props ) {
 				label: __( 'Users', 'google-site-kit' ),
 			},
 		],
-		...rows.map( ( { metrics, dimensions } ) => [
-			parseDimensionStringToDate( dimensions[ 0 ] ),
-			metrics[ 0 ].values[ 0 ],
-		] ),
+		...rows.map( ( { metrics, dimensions } ) => {
```

Comment:
It's a harmless change, but it might be worth reverting this snippet to cut down on PR noise...

Reply:
No worries, let's keep it then, it does make things a bit more explicit.